src/main/java/org/zalando/nakadi/service/converter/VersionZeroConverter.java

```diff
@@ -32,6 +43,71 @@
         return CursorConverter.Version.ZERO;
     }
 
+    public List<NakadiCursor> convertBatched(final List<SubscriptionCursorWithoutToken> cursors) throws
+            InvalidCursorException, InternalNakadiException, NoSuchEventTypeException, ServiceUnavailableException {
+        final NakadiCursor[] result = new NakadiCursor[cursors.size()];
```

Comment:
@v-stepanov In this method using arrays is more error-prone, than using collections. And for me it is really strange, that for some reason arrays usage is always treated as some kind of crime.